components/atom/badgeCounter/src/index.scss

```diff
@@ -0,0 +1,56 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$bd-badge-counter: none !default;
+$bdc-badge-counter: $c-error !default;
+$c-badge-counter: $c-white !default;
+$border-size-badge-counter-large: 6px !default;
+$size-badge-counter-large: 16px !default;
+$border-size-badge-counter-medium: 5px !default;
+$size-badge-counter-medium: 12px !default;
+$border-size-badge-counter-small: 3px !default;
+$size-badge-counter-small: 8px !default;
+
+.sui-BadgeCounter {
+  position: relative;
+
+  &-type {
+    border-color: $bdc-badge-counter;
+    border-style: solid;
+    background: $c-badge-counter;
+    border-radius: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+  }
+
+  &-type {
+    &-bullet-small {
+      border-width: $border-size-badge-counter-small;
+    }
+
+    &-bullet-medium {
+      border-width: $border-size-badge-counter-medium;
+    }
+
+    &-bullet-large {
+      border-width: $border-size-badge-counter-large;
+    }
+  }
+
+  &-size {
+    &-small {
```

Comment:
in order to be more consistent with other components, i would define classnames following SUIT naming convention: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md
for instance: 

```
.sui-BadgeCounter--small {}
.sui-BadgeCounter--bulletSmall {}
```

how does it looks to you?

Reply:
👍